server/src/services/semanticHighlight/highlighters/FunctionDefinitionHighlighter.ts

```diff
@@ -9,11 +9,11 @@ export class FunctionDefinitionHighlighter extends HighlightVisitor {
   public tokenKinds = new Set([TokenKind.Identifier]);
 
   public enter(nodeWrapper: SlangNodeWrapper): void {
-    const ancestors = nodeWrapper.pathRuleNodes;
+    const ancestors = nodeWrapper.ancestors();
     if (
       nodeWrapper.type === NodeType.Token &&
       nodeWrapper.kind === TokenKind.Identifier &&
-      ancestors[ancestors.length - 1]?.kind === RuleKind.FunctionDefinition
+      ancestors[ancestors.length - 2]?.kind === RuleKind.FunctionDefinition
```

Comment:
I wonder if this should have triggered a test failure?
Should we add a test case to `test/protocol/test/textDocument/semanticTokens/full.test.ts`?

Reply:
Thanks!
In that case, I wonder how it succeeded before. Do we need to add it to CI somehow?

Reply:
Got it.